models/product-type/src/product-type-draft/generator.ts

```diff
@@ -1,15 +1,16 @@
 import { AttributeDefinitionDraft } from '@commercetools-test-data/attribute-definition';
 import { fake, Generator } from '@commercetools-test-data/core';
-import { TProductTypeDraft } from '../types';
+import { TProductTypeDraftDefault } from '../types';
 
 // https://docs.commercetools.com/api/projects/productTypes#producttypedraft
 
-const generator = Generator<TProductTypeDraft>({
+const generator = Generator<TProductTypeDraftDefault>({
   fields: {
     name: fake((f) => f.random.words(2)),
     key: fake((f) => f.lorem.slug()),
     description: fake((f) => f.random.words()),
     attributes: fake(() => [AttributeDefinitionDraft.random()]),
+    attributeDefinitions: fake(() => [AttributeDefinitionDraft.random()]),
```

Comment:
Same question here - I'm assuming these two fields should not be the same shape.

Reply:
In this case, yes they are the same.
![image](https://user-images.githubusercontent.com/8136478/220656637-00b111d4-bb27-42df-ab3a-1209f905c063.png)
![image](https://user-images.githubusercontent.com/8136478/220656705-73b07e2d-8ae3-4afb-8d86-44630239040c.png)